### PR TITLE
fix: minor bug fixes

### PR DESCRIPTION
**Pull request by john-d-pelingo**

### CODE REVIEW

File: src/pages/transcripts.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function Transcripts({ data }) {
             <nav>
               <ol>
                 {articles.map((article) => (
-                  <li key={article.title}>
+                  <li key={article.path}>
```
Comment (nickserv) [inline src/pages/transcripts.js:26]: It seems like both the title and path properties are defined. What's the advantage to using path instead? Is it more stable or unique?

**Review by nickserv — CHANGES_REQUESTED** [review]
The transcript URL fix is great, just have one question.

File: src/pages/transcripts.js (hunk 1 above)
Comment (vcarl) [reply src/pages/transcripts.js:26]: Since this isn't apparent in the UI, either seems fine to me. Titles seem more likely to collide than paths

**Review by vcarl — APPROVED** [review]

File: src/pages/transcripts.js (hunk 1 above)
Comment (john-d-pelingo) [reply src/pages/transcripts.js:26]: > Titles seem more likely to collide than paths

Yes, this was the intended fix. On dev, it keeps popping the warning of same key values.

**Pull request merged**